src/WorkBC.Tests/Fixtures/NocFixture.cs

```diff
@@ -8,7 +8,10 @@ public static class NocFixture
         public static List<NocCode2021> NocCodes2021 =>
          new List<NocCode2021>
             {
-                new NocCode2021 {Code = "21234", Id= 21234, Title= "Web developers and programmers"},
+                new NocCode2021 {Code = "11202", Code2016 = "4163", Id= 21234, Title= "Web developers and programmers"},
```

Comment:
Please test some combinations where there are multiple NOC 2016 entries.

Reply:
Added a few unit tests for multi 2016 codes scenario.